app/controllers/laundry_rooms_controller.rb

```diff
@@ -25,4 +25,11 @@ def create_booking
       redirect_to root_path
     end
   end
+
+  def delete_booking
+    active_bookings = current_user.bookings.detect { |booking| booking.time == params[:time] }
```

Comment:
We are dealing with one perticular booking here, right? Why plural form on the variable name?

Reply:
Ok.